ktor-client/ktor-client-core/common/src/io/ktor/client/plugins/cache/HttpCache.kt

```diff
@@ -280,7 +280,7 @@ public class HttpCache private constructor(
             return null
         }
 
-        return storage.store(response)
+        return storage.store(response, response.varyKeys(), isSharedClient)
```

Comment:
do you mind deprecating `CacheStorage.store(response: HttpResponse): CachedResponseData` function with level = warning to avoid this mistake in the future, please?

Reply:
@rsinukov 
Thank you for your review!
I've added a Deprecated annotation. [f13c0c6](https://github.com/ktorio/ktor/pull/3683/commits/f13c0c603ce9bda4d79555dd418b704ce3d294f0)